lib/src/objector.dart

```diff
@@ -22,11 +22,14 @@ class Objector extends RedditBase {
     if (data.containsKey('name')) {
       // Redditor type.
       return new Redditor.parse(reddit, data);
-    } else if (data.containsKey('kind') && (data['kind'] == 't1')) {
+    } else if (data.containsKey('kind') &&
+        (data['kind'] == Reddit.defaultCommentKind)) {
```

Comment:
Are we just referring to the defaults for now or does this allow for users to specify custom 'kinds' in the config file? If so, `defaultCommentKind` probably isn't the best name.

Reply:
For now, we are keeping this unchangeable for now. In my next PR, I'll fix this before going on to MultiReddit, since this isn't difficult to do with the way I have it set up :)

Reply:
Oh, I wouldn't worry about it just yet. Can you just file an issue so we don't forget to come back to it? :)